sys/decl.go

```diff
@@ -391,6 +391,8 @@ type IntType struct {
 	TypeCommon
 	TypeSize uintptr
 	Kind     IntKind
+	IsRange  bool
+	IntRange []uintptr
```

Comment:
randInt and rand64 both return uintptr in prog/rand.go . so rangeBegin and rangeEnd should also be uintptr?  since the range may be for  int8 ,  int16, int32, not only for int64. Does the negative also be presented by  uintptr?  Could you tell me the difference between uintptr and int64 ? Thanks